staging/src/k8s.io/component-base/config/types.go

```diff
@@ -78,3 +78,18 @@ type DebuggingConfiguration struct {
 	// enableProfiling is true.
 	EnableContentionProfiling bool
 }
+
+// TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
```

Comment:
drop the internal version of this type (see comment on pkg/kubelet/apis/config/types.go for an overview of where we want component-base type snippets to live)

Reply:
this was a source of confusion for me, got it now, thanks